### DISCUSSION TITLE
Tooltip info reduction

### DISCUSSION THREAD

**Discussion by ricraftis** [RyotaUshio/obsidian-pdf-plus#157] [Q&A]
I have a colour system setup for highlighting: 
Colour and Meaning
Red - Very important, critical
Yellow - Interesting point
Green - Supporting argument/example
Orange - Disagree with author
Grey - Vocabulary, names and dates, definitions
Blue - Related, research this paper

When I hover over each colour, the tooltip is very cluttered with repetitive information. It would be really handy to set the tooltip to simply reflect each colour's meaning. Unfortunately, there is no standard for highlighting academic articles that can be adopted to my knowledge. Everyone uses one that suits them. The tooltip would help ensure the right colour is being used.

Cheers,

**Comment by RyotaUshio** [comment]
I will add a new option "Quiet tooltips in color palette" in the next version.

<img width="186" alt="image" src="https://github.com/RyotaUshio/obsidian-pdf-plus/assets/72342591/ff20666b-83d9-46ec-ac7d-ec880464ce67">

**Comment by ricraftis** [comment]
Preferably labelled same as in settings. e.g.
![Snag_4d101d7](https://github.com/RyotaUshio/obsidian-pdf-plus/assets/121541749/70cb925a-beed-4663-80c4-acaa77b08754)

**Reply by RyotaUshio** [reply]
Released in [0.39.15](https://github.com/RyotaUshio/obsidian-pdf-plus/releases/tag/0.39.15)!

**Comment by ricraftis** [comment]
Brilliant! Many thanks.
